corehq/apps/es/fake/es_query_fake.py

```diff
@@ -132,6 +134,36 @@ def _term_query(doc):
 
         return self._filtered(_term_query)
 
+    def date_range(self, field, gt=None, gte=None, lt=None, lte=None):
```

Comment:
This doesn't accept both `gt` and `lt` at the same time, right? If so, worth adding a comment indicating so.